modules/migration-configuring-azure.adoc

```diff
@@ -92,14 +107,30 @@ $ az storage container create \
 +
 [source,terminal]
 ----
-$ AZURE_SUBSCRIPTION_ID=`az account list --query '[?isDefault].id' -o tsv` \
-  AZURE_TENANT_ID=`az account list --query '[?isDefault].tenantId' -o tsv` \
-  AZURE_CLIENT_SECRET=`az ad sp create-for-rbac --name "velero" \
-  --role "Contributor" --query 'password' -o tsv` \
-  AZURE_CLIENT_ID=`az ad sp list --display-name "velero" \
-  --query '[0].appId' -o tsv`
+$ AZURE_SUBSCRIPTION_ID=`az account list --query '[?isDefault].id' -o tsv`
+  AZURE_TENANT_ID=`az account list --query '[?isDefault].tenantId' -o tsv`
+----
+
+. Create a service principal with the `Contributor` role, assigning a specific `--role` and `--scopes`:
++
+[source,terminal]
+----
+$ AZURE_CLIENT_SECRET=`az ad sp create-for-rbac --name "velero" \
+                                                --role "Contributor" \
+                                                --query 'password' -o tsv \
+                                                --scopes /subscriptions/$subscriptionID/resourceGroups/$resourceGroup`
```

Comment:
A small suggestion, As we already fetching the Azure subscriptionID in line 110 in the variable AZURE_SUBSCRIPTION_ID and the azure resourceGroup using variable AZURE_RESOURCE_GROUP , So we can use them directly here.  So please update them here.

--scopes /subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$AZURE_RESOURCE_GROUP